spec/support/feature_helpers.rb

```diff
@@ -0,0 +1,10 @@
+module FeatureHelpers
+  def start_new_application
+    visit '/'
+    click_link 'Start now'
+  end
+end
+
+RSpec.configure do |c|
+  c.include FeatureHelpers, type: :feature
+end
```

Comment:
What do we gain with this change?

Why didn't **visit applications_new_path** work?